### PR TITLE
FIX/3259/Fix wrong entrypoint of analysis docker

### PR DESCRIPTION
**Pull request by ukinimod**
# Fix wrong entrypoint of analysis docker

Issue:  #3259 

## Description

Implementation: Uses `ccsh` as entrypoint instead of `tail -f /dev/null`, thus `docker run -it -v $(pwd):/src --workdir /src codecharta/codecharta-analysis rawtextparser .` works for example.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Fixed 🐞
 
+-   Fix entrypoint for analysis docker image [#3259]
```
Comment (ce-bo) [inline CHANGELOG.md:18]:
```suggestion
-   Fix entrypoint for analysis docker image [#3259](https://github.com/MaibornWolff/codecharta/pull/3259)
```

File: analysis/Dockerfile [hunk 2]
```diff
@@ -35,4 +35,4 @@ RUN wget -qO /tmp/sonar-scanner.zip https://binaries.sonarsource.com/Distributio
     mkdir --parents /usr/local/conf ; cp ./conf/sonar-scanner.properties /usr/local/conf/; \
     cp -r ./jre/ /usr/local/jre; cd /; rm /tmp/sonar-scanner.zip; rm -rf /tmp/sonar-scanner-*-linux;
 
-ENTRYPOINT ["tail", "-f", "/dev/null"]
\ No newline at end of file
+ENTRYPOINT ["ccsh"]
```
[the remark this answers is not in the document]
Comment (ukinimod) [reply analysis/Dockerfile:38]: It is a design decision: Other commands could be run be specifying "--entrypoint" as is needed now for ccsh due to the faulty entrypoint. But, if the image is used a lot for the other tools it's probably best to have ccsh specified as command and no entrypoint at all. I will change it thus. So you would have to call:
`docker run -it -v $(pwd):/src --workdir /src codecharta/codecharta-analysis ccsh rawtextparser .`
I would also suggest adding a better description in https://hub.docker.com/r/codecharta/codecharta-analysis.

[the remark this answers is not in the document]
Comment (ce-bo) [reply analysis/Dockerfile]: I created an issue for updating the documentation. https://github.com/MaibornWolff/codecharta/issues/3304

**Review by ce-bo — APPROVED** [review]
LGTM, thanks for your support!

**Pull request merged**